### PR TITLE
Add fake API for local dev & tests

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
### Major changes

- Fake API spits out the data defined in `fake-api/data.json` with the routes configured in `fake-api/routes.json`
- Created separate web-dev-server configs (1 using fake API, 1 for prod API)
- Changed the web-test-runner config to use fake API (this should prevent flakes in the other PRs)
- Added the npm run start commands for local and k8s to visualize different data
- Updated README

### CODE REVIEW

File: web/test/testgrid-index.test.ts [hunk 1]
```diff
@@ -22,10 +22,6 @@ describe('Testgrid Index page', () => {
     element = await fixture(html`<${tag}></${tag}>`);
   });
 
-  it('passes the a11y audit', async () => {
```
Comment (michelle192837) [inline web/test/testgrid-index.test.ts:25]: QQ, any reason for removing this?

**Review by michelle192837 — APPROVED** [review]
Looks great, thanks for the change Sultan!

Holding for the one comment, otherwise good to go.

/hold

File: web/test/testgrid-index.test.ts (hunk 1 above)
Comment (michelle192837) [reply web/test/testgrid-index.test.ts:25]: Ah, gotcha! Commenting + TODO'ing sounds good to me if that's alright, I think we want to pass the accessibility checks but don't have to do it here.

Comment (sultan-duisenbay) [reply web/test/testgrid-index.test.ts:25]: Adding TODOs now

**Pull request merged**